app/models/user.rb

```diff
@@ -188,7 +188,7 @@ def self.from_omniauth(auth)
   def self.create_from_provider_data(provider_data)
     user = User.find_by email: provider_data.info.email
 
-    return user if user
+    return if user
```

Comment:
I would like to see the changes made here combined with the proposed modifications in PR #902 to make sure it handles both cases.

Reply:
I've added a new commit to include that proposed idea: https://github.com/portagenetwork/roadmap/pull/907/commits/1985f1e8e0352ad9e485a4f7a6fd6c3a68b7ebcf